### PR TITLE
tools: client: Port to the new AA API

### PR DESCRIPTION
**Pull request by sameo**
And update our dependencies while at it.

### CODE REVIEW

**Review by fitzthum — APPROVED** [review]
LGTM. I was just about to do this actually

File: tools/client/README.md [hunk 1]
```diff
@@ -14,15 +14,13 @@ For help:
 Request resource from KBS:
 
 ```shell
-./client --resource-uri kbs://127.0.0.1:8080/<repository>/<type>/<tag>
+./client --kbs-uri https://127.0.0.1:8080/ --resource-path /<repository>/<type>/<tag>
```
Comment (jialez0) [inline tools/client/README.md:17]: Do we need the `/` after `https://127.0.0.1:8080` ?

Comment (sameo) [reply tools/client/README.md]: Fixed, good catch.

**Review by jialez0 — APPROVED** [review]
LGTM.

**Pull request merged**